Makefile

```diff
@@ -16,6 +16,11 @@ endif
 
 KIND_CLUSTER ?= chart-testing
 REPOSITORY ?= gcr.io/cockroachlabs-helm-charts/cockroach-self-signer-cert
+BUNDLE_IMAGE ?= cockroach-operator-bundle # TODO: Finalise names for images, project, registry and project
+OPERATOR_IMAGE ?= cockroachdb
```

Comment:
this will have conflict with the catalog image that `opm index` command builds, whose name is also `cockroach-operator`. need to decide something distinguishing for both.

Reply:
IMO, This should be the `OPERATOR_IMAGE ?= cockroach-helm-operator`

Reply:
changed accordingly and now each image has unique name